base_import_default_enable_tracking/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Base Import Default Enable Tracking
+===================================
+
+This modules simply enables history tracking when doing an import.
```

Comment:
This is a feature that for some reason was removed from the UI, right?
Any chance that it is is an upstream bug?